src/main/java/com/faforever/api/data/checks/IsEntityOwnerFilter.java

```diff
@@ -0,0 +1,72 @@
+package com.faforever.api.data.checks;
+
+import com.faforever.api.data.domain.Login;
+import com.faforever.api.data.domain.OwnableEntity;
+import com.faforever.api.error.ProgrammingError;
+import com.faforever.api.security.ElideUser;
+import com.yahoo.elide.annotation.SecurityCheck;
+import com.yahoo.elide.core.Path;
+import com.yahoo.elide.core.Path.PathElement;
+import com.yahoo.elide.core.filter.Operator;
+import com.yahoo.elide.core.filter.expression.FilterExpression;
+import com.yahoo.elide.core.filter.predicates.FilterPredicate;
+import com.yahoo.elide.core.security.RequestScope;
+import com.yahoo.elide.core.security.checks.FilterExpressionCheck;
+import com.yahoo.elide.core.type.Field;
+import com.yahoo.elide.core.type.Type;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static com.yahoo.elide.core.type.ClassType.INTEGER_TYPE;
+
+public class IsEntityOwnerFilter {
```

Comment:
We need a JavaDoc describing when to use this Filter over the normal SecurityCheck